openmetadata-service/src/main/java/org/openmetadata/service/resources/services/ingestionpipelines/IngestionPipelineResource.java

```diff
@@ -622,9 +621,7 @@ public Response getLastIngestionLogs(
   public Response addPipelineStatus(
       @Context UriInfo uriInfo,
       @Context SecurityContext securityContext,
-      @Encoded
-          @Parameter(description = "fqn of the ingestion pipeline", schema = @Schema(type = "string"))
```

Comment:
@ulixius9 why is this parameter encoded? It should work without encoding. We need upgrade changes for this to rewrite pipeline entities fqn without url encoding, when we commit this change. cc @TeddyCr